templates/vue/src/components/tyde/TydeCockpit.vue

```diff
@@ -1,42 +1,29 @@
 <template>
-  <section id="tyde-menu-home">
-    <nav>
-      <ul>
-        <tyde-tab
-          v-for="tab in tabs"
-          :key="tab"
-          :is-active="tab === activeTab"
-          @click="setActiveTab(tab)"
-        >
-          See {{ tab }}
-        </tyde-tab>
-      </ul>
-    </nav>
-    <ul class="logs">
-      <tyde-log
-        v-for="log in visibleLogs"
-        :key="log.name"
-        :log="log"
-        @close="$emit('close')"
-      />
-    </ul>
+  <section id="tyde-cockpit" :style="cockpitImage">
+    <div v-for="node in this.nodes" :key="node.id">
+      <div
+        v-if="isNodeModuleType(node)"
```

Comment:
`v-for` with `v-if` sometimes doesn't work as expected. I would suggest adding a computed property that filters this out, e.g.

```js
modules() {
  return this.nodes.filter(isNodeModuleType)
}
```